simulator.py

```diff
@@ -464,29 +480,16 @@ def _build(self, duration_ms):
         self.machine_controller = MachineController(self.spinnaker_hostname)
         self.machine_controller.boot(self.spinnaker_width, self.spinnaker_height)
 
-        # Retrieve a machine object
-        spinnaker_machine = self.machine_controller.get_machine()
-
-        logger.debug("Found %ux%u chip machine" %
-            (spinnaker_machine.width, spinnaker_machine.height))
-
-        # If we should reserve extra cores on each chip e.g. for network tester
-        # **TODO** integrate new Rig API to make this unnecessary
-        if self.reserve_extra_cores_per_chip > 0:
-            logger.info("Reserving %u extra cores per-chip"
-                        % self.reserve_extra_cores_per_chip)
-
-            # Reserve these extra cores (above monitor) on each chip
-            reservation = slice(1, 1 + self.reserve_extra_cores_per_chip)
-            constraints.append(ReserveResourceConstraint(machine.Cores,
-                                                         reservation))
-
-        logger.info("Placing and routing")
+        # Get system info
+        system_info = self.machine_controller.get_system_info()
+        logger.debug("Found %ux%u chip machine",
```

Comment:
May be more truthful to print len(system_info) giving the actual number of chips (though perhaps I'm not realising what you're using that information for?)